backend/src/main/kotlin/de/neuefische/backend/user/UserProfileResponse.kt

```diff
@@ -0,0 +1,9 @@
+package de.neuefische.backend.user
+
+data class UserProfileResponse(
+    val id: String,
```

Comment:
Same question about the id Object vs. String

Reply:
in this case the response is a DTO so it's string for the benefit of the client